composer.json

```diff
@@ -10,7 +10,9 @@
     "require": {
         "php": ">=7.3",
         "squizlabs/php_codesniffer": "^3.6",
-        "webonyx/graphql-php": "^14.9"
+        "webonyx/graphql-php": "^14.9",
+        "ext-simplexml": "*",
```

Comment:
Is that possible to avoid additional dependencies by using regex instead of XML queries?

Reply:
Isn't Simplexml bundle in 99,99% of PHP installations? I could try to do it with regexps, but it would be way harder both to create and to read.